src/trap.c

```diff
@@ -4959,7 +4959,7 @@ help_monster_out(
     if (uprob) {
         You("try to grab %s, but cannot get a firm grasp.", mon_nam(mtmp));
         if (mtmp->msleeping) {
-            wakeup(mtmp, FALSE);
+            wakeup(mtmp, FALSE, FALSE);
```

Comment:
If the monster's name is being printed here and you're trying to grab it, why would it not be revealed? Ditto for the call a few lines below.